.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -34,9 +34,9 @@ Steps to reproduce the behavior:
  - OS Build:
  - Architecture:
  - Application Version:
- - Region: 
- - Dev Version Installed: 
```

Comment:
nit: We should be consistent with spaces after colons in this list (appears to be mixed).

Reply:
We are being consistent.  This removed the trailing spaces that two of them had.  The PowerShell code that we request users to run ensures that the spacing after the colons remains consistent.